.changelog/_1563.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+server: **(Enterprise only)** Namespace deletion will now attempt to delete as many config entries as possible instead of halting on the first deletion that failed.
```

Comment:
```suggestion
server: **(Enterprise only)** Namespace deletion will now attempt to delete as many namespaced config entries as possible instead of halting on the first deletion that failed.
```